js/mexc.js

```diff
@@ -687,6 +688,11 @@ module.exports = class mexc extends Exchange {
         return this.parseTickers (data, symbols);
     }
 
+    async fetchTickersByType (type, symbols = undefined, params = {}) {
```

Comment:
We could just make it a base method, cause it's the same implementation everywhere.

Reply:
It only exists on exchanges that use a default type, so we would need to create some sort of check for default types in order to do this, or else it would be misleading on an exchange like FTX, which doesn't use defaultTypes.

Reply:
@kroitor Do you still want me to make it a base method?

Reply:
@samgermain i'll merge it as is, and we will unify it in the base classes later. Let's consider it "experimental".

Reply:
On a second thought, maybe we should not add the "ByType" methods, cause they're literally one-liners and we would have to add and maintain the other methods as well. I would put it aside for now since it's not mission-critical.